.travis.yml

```diff
@@ -80,7 +80,8 @@ script:
         --browser-arg-int build  "$TRAVIS_BUILD_NUMBER" \
         --browser-arg-int idleTimeout 300 \
         --browser-arg name "$TRAVIS_REPO_SLUG $TRAVIS_BRANCH $TRAVIS_COMMIT" \
-        --browser-arg browser "$BROWSER"
+        --browser-arg browser "$BROWSER" \
+        --browser-arg platform "Linux"
```

Comment:
As you mentioned...I think it's fine in this PR.  Maybe we'll want to separate via commits before merging.